tests/test_cover.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
```

Comment:
I don't think we're using this file anymore. We should just drop it altogether

Reply:
Plus, scripts should go into `contrib`

Reply:
Oh didn't know that, will drop it thanks

Reply:
I see this is being user here, would you like me to remove test-cover from make or let the file be https://github.com/cosmos/cosmos-sdk/blob/bc88c768185f1a70dc0f9d0f6a6dc38c96e11d7c/Makefile#L198

Reply:
Mmm... let's move into `contrib` for now. Thanks for spotting this!